src/ec/packages/ec-component-button/__snapshots__/button.test.js.snap

```diff
@@ -1,87 +1,147 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`EC - Button CTA button - icon after renders correctly 1`] = `
-"<button class=\\"ecl-button ecl-button--call\\" type=\\"submit\\">
-  <span class=\\"ecl-button__container\\">
-    <span class=\\"ecl-button__label\\">CTA Button with icon</span>
-
+<button
+  class="ecl-button ecl-button--call"
+  type="submit"
+>
+  <span
```

Comment:
this is how jest serializes DOM nodes in snapshots

the output is indented only for the purpose to bring more clarity and make it easier to compare snapshots

Reply:
Looks ok to me, an attribute per line adds clarity. The closing brackets on new lines are like prettier, I like this display.

Reply:
Actually: could it be that's it is prettier changing that in the commit hook?

Reply:
prettier doesn't run on *.snap files 😉